gulpfile.babel.js

```diff
@@ -208,6 +208,14 @@ gulp.task('default', ['clean'], () => {
   );
 });
 
+//Generates new JWT
+gulp.task('createNewToken',[],() => {
+  let jwtSecret = config.jwtSecret;
+  let jwt = require('jsonwebtoken');
+  let token = jwt.sign({data : {name: 'foo',password: 'bar'}},jwtSecret , { expiresIn: '1h' });
```

Comment:
Not sure how can these be fetched from the environment.
Are you suggesting adding below object in another file?
data : {
  name : 'foo',
 password : 'bar'
}